cli/medperf/commands/result/result.py

```diff
@@ -56,3 +59,24 @@ def list(
 ):
     """List results stored locally and remotely from the user"""
     ResultsList.run(local, mine)
+
+
+@app.command("view")
+@clean_except
+def view(
+    entity_id: Optional[int] = typer.Argument(None, help="Result ID"),
+    format: str = typer.Option(
+        "yaml",
+        "-f",
+        "--format",
+        help="Format to display contents. Available formats: [yaml, json]",
+    ),
+    local: bool = typer.Option(False, "--local", help="Display local results"),
+    mine: bool = typer.Option(False, "--mine", help="Display current-user results"),
+    output: str = typer.Option(
+        None, "--output", "-o", help="Output file to store contents"
```

Comment:
also here, I think we should add: if not provided, the output will be displayed
in all four typer commands